scrooge-sbt-plugin/src/sbt-test/scrooge/basic/build.sbt

```diff
@@ -0,0 +1,10 @@
+
+scalaVersion := "2.11.8"
+
+enablePlugins(BuildInfoPlugin)
+
+libraryDependencies ++= Seq(
```

Comment:
these versions not correct when we are doing development. can we share the code thats in `Build.scala` for setting these?